src/components/radio/Radio.js

```diff
@@ -158,12 +158,26 @@ export default class RadioComponent extends Field {
 
       this.refs.wrapper.forEach((wrapper, index) => {
         const input = this.refs.input[index];
+        switch (input.type) {
+          case 'radio' :
         if (input && input.value.toString() === value.toString()) {
           //add class to container when selected
           this.addClass(wrapper, optionSelectedClass);
         }
         else {
           this.removeClass(wrapper, optionSelectedClass);
+        }
+            break;
+          case 'checkbox' :
```

Comment:
When would a "radio" ever be a checkbox?  I believe this should be using the Select Boxes component instead.

Reply:
this is to support SelectBoxes component
`export default class SelectBoxesComponent extends RadioComponent`